fixtures/networks/butterfly.go

```diff
@@ -67,7 +67,7 @@ func ButterflySnapNet() *NetworkConf {
 				UpgradeTuktukHeight:                  -28,
 				UpgradeTuktukPowerRampDurationEpochs: builtin.EpochsInYear,
 				UpgradeTeepHeight:                    100,
-				UpgradeTockHeight:                    200,
```

Comment:
@simlecode @asamuj I don't think you meant to delete this

Reply:
Moved here: https://github.com/filecoin-project/venus/pull/6468/files/1786cc17af8ea75a4229ea36c1297913bc52ed2d#diff-5343415fb3da9697bbbf4df0167032d2132c88b2a230347a2f673ab14ca33fa3R88

Reply:
ACK